### PR TITLE
Update app and dev dependencies

### PR DESCRIPTION
**Pull request by blikblum**
Update app (BB, Mn, Underscore, Radio) and dev dependencies (jsdom, require). 
Adapt code to work with new library versions

### CODE REVIEW

**Comment by blikblum** [comment]
The tests through node/phantomjs are failing but on browser all tests pass

**Comment by blikblum** [comment]
Removed jsdom dependency since is not being used

File: bower.json [hunk 1]
```diff
@@ -17,14 +17,14 @@
   "dependencies": {
```
Comment (denar90) [inline bower.json]: @blikblum do you know why we don't have backbone as dependency here?

Comment (blikblum) [inline bower.json]: It's in bower. 
Basically npm stores dev dependencies and bower client dependencies. 

The tests uses customized BB and Mn builds.

For today standard is pretty convoluted but at the time the project started that was OK.

I changed the minimum to get working in decent shape in a modern env. Further improvements can be done later.

**Pull request merged**